mkosi.md

```diff
@@ -380,61 +392,66 @@ details see the table below.
-  is smaller. If not specified, and `mkosi.workspace/` exists, it is
-  used for this purpose. If it doesn't exist and `$TMPDIR` is set, the
-  specified directory is used. If that's not set either, `/var/tmp/`
-  is used. The data in the directory is removed automatically after
-  each build. It's safe to manually remove the contents of this
-  directories should an `mkosi` invocation be aborted abnormally (for
-  example, due to reboot/power failure). If the `btrfs` output modes
-  are selected this directory must be backed by `btrfs` too.
-
-`--force`, `-f`
+  the full OS image, though in most modes the actually used disk space
+  is smaller. If not specified, and `mkosi.workspace/` exists in the
+  local directory, it is used for this purpose. Otherwise, a
+  subdirectory in the temporary storage area is used (`$TMPDIR` if
+  set, `/var/tmp/` otherwise).
+
+: The data in the directory is removed automatically after each
+  build. It's safe to manually remove the contents of this directory
+  should an `mkosi` invocation be aborted abnormally (for example, due
+  to reboot/power failure). If the `btrfs` output modes are selected
+  this directory must be backed by `btrfs` too.
+
+`Force=`, `--force`, `-f`
 
 : Replace the output file if it already exists, when building an
   image. By default when building an image and an output artifact
-  already exists `mkosi` will refuse operation. Specify `-f` to delete
-  all build artifacts from a previous run before re-building the
-  image. If incremental builds are enabled (see below), specifying
-  this option twice will ensure the intermediary cache files are
-  removed, too, before the re-build is initiated. If a package cache
-  is used (see below), specifying this option thrice will ensure the
-  package cache is removed too, before the re-build is initiated. For
-  the `clean` operation `-f` has a slightly different effect: by
-  default the verb will only remove build artifacts from a previous
-  run, when specified once the incremental cache files are deleted
-  too, and when specified twice the package cache is also removed.
-
-`--gpt-first-lba`
+  already exists `mkosi` will refuse operation. Specify this option
+  once to delete all build artifacts from a previous run before
+  re-building the image. If incremental builds are enabled,
+  specifying this option twice will ensure the intermediary
+  cache files are removed, too, before the re-build is initiated. If a
+  package cache is used (also see the "Files" section below),
+  specifying this option thrice will ensure the package cache is
+  removed too, before the re-build is initiated. For the `clean`
+  operation this option has a slightly different effect: by default
+  the verb will only remove build artifacts from a previous run, when
+  specified once the incremental cache files are deleted too, and when
+  specified twice the package cache is also removed.
+
+  <!--  FIXME: allow `Force=<n>` -->
```

Comment:
I'm wondering whether we should do Force in config files at all. This seems like one of those options that's better off as a CLI only option. Same with --incremental.

General rant: We should get rid of the argparse hack and generate the config parser and argparse object from the `CommandLineArguments` dataclass we have now (which we should rename to just `Args` or something similar.

Reply:
> I'm wondering whether we should do Force in config files at all. This seems like one of those options that's better off as a CLI only option. Same with --incremental.

I agree, Force in the config file is weird. Maybe at least drop from the docs, and keep it in the code for compat?